src/Database/Schema/PostgresSchema.php

```diff
@@ -334,7 +334,10 @@ public function columnSql(Table $table, $name)
             'binary' => ' BYTEA',
             'float' => ' FLOAT',
             'decimal' => ' DECIMAL',
+            'tinytext' => ' TEXT',
```

Comment:
I think it would be good to match the sizes. In this case by using `varchar(some size)`